src/layers/core/screen-grid-layer/screen-grid-layer.js

```diff
@@ -79,10 +79,13 @@ export default class ScreenGridLayer extends Layer {
   draw({uniforms}) {
     const {minColor, maxColor} = this.props;
     const {model, cellScale, maxCount} = this.state;
-    const {gl} = this.context;
-    gl.depthMask(true);
     uniforms = Object.assign({}, uniforms, {minColor, maxColor, cellScale, maxCount});
-    model.render(uniforms);
+    model.draw({
```

Comment:
@1chandu Is there a reason we start using model.draw() here? We always use model.render() in deck.gl before.